definitions/containerization.md

```diff
@@ -1,12 +1,18 @@
 ---
 title: Containerization
-status: Feedback Appreciated
-category: concept
+status: Completed
+category: Technology
 ---
 ## Containerization
 
-Containerization is the process of bundling an application and its dependencies into a container image. The container build process requires adherence to the [Open Container Initiative] (https://opencontainers.org) (OCI) standard. As long as the output is a container image that adheres to this standard, which containerization tool is used does not matter.
+### What it is
 
-Before containers became prevalent, organizations relied on virtual machines (VMs) to orchestrate multiple applications on a single bare-metal machine. VMs are significantly larger than containers and require a hypervisor to run. Due to the storage, backup, and transfer of these larger VM templates, creating the VM templates is also slow.
+Containerization is the process of bundling an application and its dependencies into a [container image](https://github.com/cncf/glossary/blob/main/definitions/container-image.md). The container build process requires adherence to the [Open Container Initiative](https://opencontainers.org) (OCI) standard. As long as the output is a container image that adheres to this standard, which containerization tool is used doesn't matter.
 
-Compared to VMs, container images are lightweight. The containerization process requires a file with a list of dependencies. This file can be version controlled the build process can be automated, allowing an organization to focus on other priorities while the automated processes take care of the build.
\ No newline at end of file
+### The problem it addresses 
+
+Before containers became prevalent, organizations relied on virtual machines (VMs) to orchestrate multiple applications on a single [bare-metal machine](https://github.com/cncf/glossary/blob/main/definitions/bare_metal_machine.md). VMs are significantly larger than containers and require a hypervisor to run. Due to the storage, backup, and transfer of these larger VM templates, creating the VM templates is also slow.
```

Comment:
Another problem that containers address is that of mutability, I think it's relevant to highlight that containers are immutable, and the reliability and security guarantees that very relevant quality provide.

The problem being mutability and the ambiguity in terms of system state and reproducibility of that state, and the solution below immutability and predictable, known, versioned system states.